WordPress/Classes/ViewRelated/NUX/DomainSuggestionsButtonViewToggle.swift

```diff
@@ -0,0 +1,40 @@
+import UIKit
+
+/// UIViewController subclasses who has a bottom view that should have a hiding
+/// ability can conform to this protocol so showing/hiding is already handled.
+protocol DomainSuggestionsButtonViewToggle {
```

Comment:
Thank you for renaming this. My only suggestion is: I think Toggle is unnecessary. Maybe just make it `DomainSuggestionButtonView`? I know it's still VC specific, but I think it's ok to have it that way for now, and it can be renamed later if it is used for something else.

Reply:
I've updated this as DomainSuggestionButtonPresenter